prov/rxm/src/rxm.h

```diff
@@ -72,13 +72,16 @@
 
 #define RXM_IOV_LIMIT 4
 
-#define RXM_MR_MODES	(OFI_MR_BASIC_MAP | FI_MR_LOCAL)
+#define RXM_MR_MODES	(OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_RAW)
```

Comment:
If rxm is intended to pass this mr mode through, it must also implement the mr_map_raw/mr_unmap_key/mr_raw_attr interfaces.  Otherwise, apps can't use RMA directly.